databricks/koalas/config.py

```diff
@@ -28,33 +28,87 @@
 __all__ = ['get_option', 'set_option', 'reset_option']
 
 
-# dict to store registered options and their default values (key -> default).
-_registered_options = {
-    # This sets the maximum number of rows koalas should output when printing out various output.
-    # For example, this value determines whether the repr() for a dataframe prints out fully or
-    # just a truncated repr.
-    "display.max_rows": 1000,  # TODO: None should support unlimited.
-
-    # `plotting.max_rows` sets the visual limit on TopN plots. If it is set to 1000, the first 1000
-    # data points will be used for plotting, and this number will be seen on the right up corner.
-    "plotting.max_rows": 1000,
-
-    # 'compute.max_rows sets the limit of the current DataFrame. Set `None` to unlimit
-    # the input length. When the limit is set, it is executed by the shortcut by collecting
-    # the data into driver side, and then using pandas API. If the limit is unset,
-    # the operation is executed by PySpark. Default is 1000.
-    "compute.max_rows": 1000,  # TODO: None should support unlimited.
-
-    # This determines whether or not to operate between two different dataframs.
-    # For example, 'combine_frames' function internally performs a join operation which can be
-    # expensive in general.
-    # So, if `compute.ops_on_diff_frames` variable is not True, that method throws an exception.
-    "compute.ops_on_diff_frames": False,
-
-    # This sets the default index type: sequence, distributed and distributed-sequence.
-    "compute.default_index_type": "sequence",
-}  # type: Dict[str, Any]
-
+class Option:
```

Comment:
I am going to add docstring later tomorrow.

Reply:
you could also do it in separate PR ^^